### DISCUSSION TITLE
Missing Signage Points metric

### DISCUSSION THREAD

**Discussion by laroy-sh** [Chia-Network/chia-blockchain#16464] [Support]
I thought this MSP metric is supposed to reset automatically every 24 hours. It does not, and I'm uncertain whether it's a bug or not. What does it mean if I have a total of 75 missed signage points?

**Comment by BrandtH22** [comment]
Hey @laroyru , there is an option to manually reset the missing signage points parameter on that screen.
Can you try resetting and noting over the next day or two as to whether it continually increases?

Note - most missing signage points are due to network or system issues when they occur in low numbers (such as 75). 
Many users have experienced this when their ISP assigns a new IP or when they have for other reasons dropped their peers and have had to reconnect. Having a few missing signage points should not be a concern but if that number drastically increases it is worth debugging and reviewing deeper.

Side note - we are generally able to provide more timely and thorough support in our discord server (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
